docs/io/visualization/how_to_liv_plot.ipynb

```diff
@@ -117,6 +117,32 @@
     "plotter.generate_plot_mpl(packets_mode=\"real\")"
```

Comment:
<p>is this intentional or a typo?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>It is a typo, I was testing the output and forgot to change it back, thanks for pointing it out.</p>